wren-ui/src/apollo/server/services/askingService.ts

```diff
@@ -187,6 +192,10 @@ class BackgroundTracker {
 
           // remove the task from tracker if it is finalized
           if (isFinalized(result.status)) {
+            this.telemetry.send_event('question_answered', {
+              question: threadResponse.question,
+              result,
```

Comment:
![image](https://github.com/Canner/WrenAI/assets/1208829/ba126802-68f3-4f25-b4c7-18a187da50a7)

`Results` is like this. No personal information included checked.